src/app/authentication/authentication.component.scss

```
@@ -0,0 +1,32 @@
+@import '../../styles.scss';
```

Delete you only need and try to use absolute paths
```suggestion
@import '../../_colors.scss';
```